org.eclipse.scout.rt.chart.ui.html/src/main/resources/org/eclipse/scout/rt/chart/ui/html/json/defaultValues.json

```diff
@@ -0,0 +1,50 @@
+/**
+ * Note: This file is not a well-formed JSON file according to RFC 4627 because of the comments.
+ * You can remove the comments (and thus create a compliant JSON file again) with
+ * JsonUtility.stripJsonComments(). If the file is served by StaticResourceRequestInterceptor,
+ * this is done automatically.
+ */
+{
+  /* *** Default value definitions *** */
+  /* (Note: Array values are always tested without considering the order) */
+  "defaults": {
+    "Chart": {
+      "~data": {
+        "axes": []
+      },
+      "~config": {
+        "autoColor": true,
+        "maxSegments": 5,
+        "clickable": false,
+        "animated": true,
```

Comment:
animated does not seem to exist anymore, it is now controlled by animation: duration